ui/app/App.js

```diff
@@ -386,6 +395,46 @@ class App extends React.Component {
 
     this.Clipboard.save = () => {};
 
+    this.Config = {};
+
+    this.Config.path = (path) => {
+      const { Config = {} } = this.state;
+      let object = { Config };
+      const steps = path.split('/');
+      while (steps.length > 0) {
+        const step = steps.shift();
+        if (object[step] === undefined) {
+          object[step] = {};
+        }
+        object = object[step];
+      }
+      return object;
+    };
+
+    this.Config.update = async () => {
```

Comment:
Basically this change allows the worker configuration to be updated rather than compiled in?

Reply:
Yeah, that's the idea. This bit handles persistent storage of the configuration, the actual communication is via sys/attach